### PR TITLE
[NUI] Open ScrollableBase APIs

### PR DESCRIPTION
**Pull request by neostom432**
### Description of Change ###
Open ScrollalbleBase APIs

### API Changes ###
 - ACR: https://code.sec.samsung.net/jira/browse/TCSACR-343

### CODE REVIEW

File: src/Tizen.NUI.Components/Controls/ScrollableBase.cs [hunk 1]
```diff
@@ -251,78 +244,69 @@ public bool ScrollEnabled
...
         public bool SnapToPage { set; get; } = false;
 
         /// <summary>
-        /// [Draft] Get current page.
+        /// Get current page.
         /// Working propery with SnapToPage property.
         /// </summary>
-        /// This may be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API
-        [EditorBrowsable(EditorBrowsableState.Never)]
+        /// <since_tizen> 8 </since_tizen>
```
Comment (m-macias) [inline src/Tizen.NUI.Components/Controls/ScrollableBase.cs:257]: /// Working propery with SnapToPage property.

Spelling.

Comment (neostom432) [reply src/Tizen.NUI.Components/Controls/ScrollableBase.cs:257]: DONE

File: src/Tizen.NUI/src/public/BackKeyEventSignal.cs [hunk 2]
```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright(c) 2019 Samsung Electronics Co., Ltd.
```
Comment (rabbitfor) [inline src/Tizen.NUI/src/public/BackKeyEventSignal.cs:2]: 2020

File: src/Tizen.NUI/src/public/BackKeyManager.cs [hunk 3]
```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright(c) 2019 Samsung Electronics Co., Ltd.
```
Comment (rabbitfor) [inline src/Tizen.NUI/src/public/BackKeyManager.cs:2]: ditto

File: src/Tizen.NUI/src/public/BackKeyEventSignal.cs (hunk 2 above)
Comment (neostom432) [reply src/Tizen.NUI/src/public/BackKeyEventSignal.cs]: It is not supposed to merge. Deleted.

File: src/Tizen.NUI/src/public/BackKeyManager.cs (hunk 3 above)
Comment (neostom432) [reply src/Tizen.NUI/src/public/BackKeyManager.cs]: It is not supposed to merge. Deleted.

**Pull request merged**